### ISSUE TITLE
Colorblind Mode

### ISSUE THREAD

**Issue by danielzeljko** [LangCorrect/react-client-mui#49]

**Comment by KalebArmbrust** [comment]
@danielzeljko Hello is this Open Source? If so I would love to help out with this issue!

**Comment by danielzeljko** [comment]
@KalebArmbrust

Yes, it is! I'll have to figure out how to get the MIT License added as it does not show up on the sidebar for some reason.

![image](https://github.com/LangCorrect/react-client-mui/assets/115326106/d82a3b64-b3fa-433d-b1b2-f553f25ca503)

Thank you for deciding to contribute. Happy to have you onboard. I have assigned the ticket to you. Please let me know if you have any questions.

**Comment by KalebArmbrust** [comment]
@danielzeljko Great, thank you! What about instead of the cog and popup, we add a toggle button that says "Color-Blind Mode" that changes the text color?

**Comment by danielzeljko** [comment]
@KalebArmbrust

I don't expect the colorblind mode to be used frequently enough to always be visible (it's basically a set once-and-forget setting), so it might be better to just have the toggle be inside of a settings popup. This will also make the user interface clean and uncluttered (mobile), especially if the colorblind mode is just one of many settings available. I may be missing something, so can you please tell me why you originally wanted to make it always visible?

**Comment by KalebArmbrust** [comment]
@danielzeljko I have gotten rather busy and do not have time for this issue right now. I apologize, please unassign me. Best of luck!

**Comment by danielzeljko** [comment]
@KalebArmbrust No worries. I have unassigned you from the issue. If you ever feel like contributing again you are more than welcome!

**Comment by Kaleshe** [comment]
Hi @danielzeljko , I'd like to take this on please
